canonical-websites/snapcraft.io
# Add authentication system

# Summary

Add authentication system in snapcraft.io

# QA 

- `./run`
- http://0.0.0.0:8004/account
- You should be redirected to login.ubuntu.com
- Once you logged in you should be redirected to the account page
- Reload the page, you shouldn't be redirected to the login page
- http://0.0.0.0:8004/logout
- You should be redirected to /
- Try to access http://0.0.0.0:8004/account
- You should be redirected to the login page
